### PR TITLE
Ruby 3.1 is now the default

### PR DESCRIPTION
**Pull request by abitrolly**
Updating docs after for https://github.com/gitpod-io/workspace-images/pull/889

## How to test
```
ruby --version
```

## Documentation

Does this PR require updates to the documentation at www.gitpod.io/docs?
* Yes
  * 1. Please create a docs issue: https://github.com/gitpod-io/website/issues/new?labels=documentation&template=DOCS-NEW-FEATURE.yml&title=%5BDocs+-+New+Feature%5D%3A+%3Cyour+feature+name+here%3E

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2836"></a>

### CODE REVIEW

**Comment by filiptronicek** [comment]
Relevant page: https://www.gitpod.io/docs/introduction/languages/ruby#ruby-in-gitpod

**Comment by abitrolly** [comment]
This block gives `DL3059 info: Multiple consecutive `RUN` instructions. Consider consolidation.` warning when validating with `hadolint`.
```
RUN echo "rvm_gems_path=/home/gitpod/.rvm" > ~/.rvmrc
RUN bash -lc "rvm install ruby-3.1.1 && \
              rvm use ruby-ruby-3.1.1 --default"
              rvm use ruby-3.1.1 --default"
RUN echo "rvm_gems_path=/workspace/.rvm" > ~/.rvmrc
```
The advice is to merge the lines https://github.com/hadolint/hadolint/wiki/DL3059. Or maybe the block could be placed into separate `switch-ruby.sh` script to get an oneliner here.

**Comment by axonasif** [comment]
@abitrolly in here were doing it that way:
https://github.com/gitpod-io/template-ruby-on-rails-postgres/blob/main/.gitpod.Dockerfile

File: gitpod/docs/introduction/languages/ruby.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ USER gitpod
 # Install Ruby version 3.1.1 and set it as default
 RUN echo "rvm_gems_path=/home/gitpod/.rvm" > ~/.rvmrc
 RUN bash -lc "rvm install ruby-3.1.1 && \
-              rvm use ruby-ruby-3.1.1 --default"
+              rvm use ruby-3.1.1 --default"
 RUN echo "rvm_gems_path=/workspace/.rvm" > ~/.rvmrc
```
Comment (axonasif) [inline gitpod/docs/introduction/languages/ruby.md:35]: RUN version=ruby-3.1.1 \
    && printf 'rvm_gems_path=%s\n' "$HOME/.rvm" > $HOME/.rvmrc \
    && bash -lc "rvm reinstall ruby-${version} && rvm use ruby-${version} --default && gem install rails" \
    && printf 'rvm_gems_path=%s\n' "/workspace/.rvm" > $HOME/.rvmrc \
    && printf '{ rvm use $(rvm current); } >/dev/null 2>&1\n' >> "$HOME/.bashrc.d/70-ruby"

Comment (abitrolly) [reply gitpod/docs/introduction/languages/ruby.md:35]: Gives me the error while applying suggestion.

image

Maybe you can directly edit the file? https://github.com/abitrolly/website-1/edit/patch-1/gitpod/docs/introduction/languages/ruby.md?pr=%2Fgitpod-io%2Fwebsite%2Fpull%2F2836 If not, I can try to commit it separately.

Comment (axonasif) [reply gitpod/docs/introduction/languages/ruby.md:35]: Feel free to do it 🙌

Comment (abitrolly) [reply gitpod/docs/introduction/languages/ruby.md]: Committed. Removed "install rails" from the end and reformatted a bit. Now I need to test that it actually works. )

Comment (abitrolly) [reply gitpod/docs/introduction/languages/ruby.md]: Tested it works.

**Review by axonasif — APPROVED** [review]
Looks good!

Are you on our [discord server](https://gitpod.io/chat) as well?

**Comment by abitrolly** [comment]
> Are you on our [discord server](https://gitpod.io/chat) as well?

Nope. Should I? There are so many distracting chats that I do not check them very often

**Comment by abitrolly** [comment]
Joined. )

Speaking of appreciation, Ii you can add 'hacktoberfest-accepted` label to this PR, that would help me score some point in one ongoing event.

**Comment by axonasif** [comment]
@abitrolly it might not be necessary 😉 
Screenshot 2022-10-06 at 10 30 43 AM

**Comment by abitrolly** [comment]
@axonasif that rule comes after this one. )

image

**Comment by axonasif** [comment]
Hmm, but we're not participating in hacktoberfest to add the topic "hacktoberfest" on this repo.
Is adding "hactoberfest-accepted" to a PR enough?

**Comment by abitrolly** [comment]
@axonasif yes, I am pretty sure that OR clause is designed specifically for the cases when maintainers don't want to attract too much attention to their repos during that time.

**Comment by axonasif** [comment]
Hey @abitrolly, we decided that we will not be adding the label since we're not participating repository wide anyway.

**Comment by abitrolly** [comment]
At least I've tried. :D

**Pull request merged**